velox/expression/tests/ExpressionFuzzer.cpp

```diff
@@ -710,6 +721,13 @@ ExpressionFuzzer::ExpressionFuzzer(
   // Register function override (for cases where we want to restrict the types
   // or parameters we pass to functions).
   registerFuncOverride(&ExpressionFuzzer::generateSwitchArgs, "switch");
+  registerFuncOverride(
```

Comment:
We don't want to hard-code input generators for specific functions in the core Fuzzer. These generators depend on the dialect and might be different between Presto and Spark. 

Maybe follow AggregationFuzzer which provides an InputGenerator interface and takes a map from function name to an instance of the generator. This way each dialect can provide their own generators and the logic is decoupled from the Fuzzer itself.

What do you think?

Reply:
Thanks for your suggestion. Added a map of ArgumentGenerator to decouple functions with the fuzzer.